README.md

```diff
@@ -1,12 +1,16 @@
 # TOPMed Analysis Pipeline — WDL Version
 
 [![WDL 1.0 shield](https://img.shields.io/badge/WDL-1.0-lightgrey.svg)](https://github.com/openwdl/wdl/blob/main/versions/1.0/SPEC.md)  
-This is a work-in-progress project to implement some components of the University of Washington [TOPMed pipeline](https://github.com/UW-GAC/analysis_pipeline) into Workflow Description Language (WDL) in a way that closely mimics [the CWL version of the UW Pipeline](https://github.com/UW-GAC/analysis_pipeline_cwl). In other words, this is a WDL that mimics a CWL that mimics a Python pipeline. All three pipelines use the same underlying R scripts which do most of the heavy lifting, making their results directly comparable.
+This project is a Workflow Description Language (WDL) implementation of several components of the University of Washington [TOPMed pipeline](https://github.com/UW-GAC/analysis_pipeline), purposely done in a way that closely mimics [the CWL version of the UW Pipeline](https://github.com/UW-GAC/analysis_pipeline_cwl). In other words, this is a WDL that mimics a CWL that mimics a Python pipeline. All three pipelines use the same underlying R scripts which do most of the actual analysis, making their results directly comparable.
 
 ## Features
 * This pipeline is very similar to the CWL version, and while the main differences between the two [are documented](https://github.com/DataBiosphere/analysis_pipeline_WDL/blob/main/_documentation_/for%20users/cwl-vs-wdl-user.md), testing indicates they are functionally equivalent -- so much so that files generated by the CWL are used as truth files for the WDL   
 * As it works in a Docker container, it does not have any external dependencies other than the usual setup required for [WDL](https://software.broadinstitute.org/wdl/documentation/quickstart) and [Cromwell](http://cromwell.readthedocs.io/en/develop/)
 * Contains multiple checker workflows for validating sets of known inputs and expected outputs
+* Open-access sample data is provided, based upon sample data provided by UWGAC, itself based upon 1000 Genomes data
```

Comment:
Is the sample data we provide "based" upon the sample data provided by UWGAC? Or is it basically the same? I don't remember. If it is so, maybe reframe it to reflect that?

Suggestion:

Open-access sample data is provided, based upon sample data provided by UWGAC. This sample data is based upon the 1000 Genomes data

Reply:
Some of the data is modified. For example, UWGAC's copy of either chrX or chr1 (can't remember) has malformed data inside of it and breaks on either vcf to gds or ld pruning (again, can't remember) in both CWL and WDL form.